packages/@react-spectrum/autocomplete/src/MobileSearchAutocomplete.tsx

```diff
@@ -227,7 +227,7 @@ const SearchAutocompleteButton = React.forwardRef(function SearchAutocompleteBut
 
   return (
     <div
-      {...mergeProps(hoverProps, focusProps, buttonProps)}
+      {...mergeProps(buttonProps, focusProps, hoverProps)}
```

Comment:
this got fixed in https://github.com/adobe/react-spectrum/pull/3801
please revert this file